frontend/app/mint/[input_index]/page.tsx

```diff
@@ -0,0 +1,361 @@
+
+  const userAddress = score.user_address?.toLowerCase();
+
+  const mint = async () => {
+    if (score?._proof == undefined) {
+      alert("No proofs yet.");
+      return;
+    }
+    if (!nftContract) {
+      alert("Contract not loaded.");
+      return;
+    }
+    if (alreadyMinted) {
+      alert("Already Minted.");
+      return;
+    }
+    if (operator == userAddress && (!gamelogOwner || gamelogOwner == '0x0000000000000000000000000000000000000000')) {
+      alert("Operator generated (register first).");
+      return;
+    }
+    nftContract.mint(score._payload,score._proof).
+      then((res: any) => {
+        res.wait(1).then(
+          (receipt: any) => {
+            console.log(receipt)
+            reload("Nft Minted!");
+          }
+        );
+      }
+    );
+  };
+
+  const register = async () => {
+    if (!signature || signature.length != 132) {
+      alert("Invalid signature.");
+      return;
+    }
+    if (!nftContract) {
+      alert("Contract not loaded.");
+      return;
+    }
+    if (alreadyMinted) {
+      alert("Already Minted.");
+      return;
+    }
+    if (signerAddress == operator){
+      alert("Operator can't register.");
+      return;
+    }
+    if (gamelogOwner != '0x0000000000000000000000000000000000000000') {
+      alert("Already registered.");
+      return;
+    }
+    const r = signature.slice(0, 66);
+    const s = "0x" + signature.slice(66, 130);
+    const v = parseInt(signature.slice(130, 132), 16);
+
+    nftContract.setGameplayOwner(score.gameplay_hash,r,s,v).
+      then((res: any) => {
+        res.wait(1).then(
+          (receipt: any) => {
+            console.log(receipt)
+            reload("Gameplay Registered");
+          }
+        );
+      }
+    );
+  };
+
+  let mintEnabled = false;
+  let mintMessage = <></>;
+  if (score?._proof == undefined)
+    mintMessage = <span>(check again later, no proofs yet)</span>;
+  else if (!nftContract)
+    mintMessage = <span>(connect wallet)</span>;
+  else if (alreadyMinted)
+    mintMessage = <span>(already Minted)</span>;
+  else if (operator == userAddress && (!gamelogOwner || gamelogOwner == '0x0000000000000000000000000000000000000000'))
+    mintMessage = <span>(operator generated, register first)</span>;
+  else {
+    mintEnabled = true;
+    mintMessage = <span></span>;
+  }
+
+  const mintButton =
+    <button className="btn" onClick={() => {mint()}} disabled={!mintEnabled}>
+      <span>Mint Score Screenshot NFT</span><br/>
+      {mintMessage}
+    </button>
+  ;
+
+  let showRegister = false;
+  let registerEnabled = false;
+  let registerMessage = <></>;
+  if (!nftContract)
+    registerMessage = <span>(connect wallet)</span>;
+  else if (alreadyMinted)
+    mintMessage = <span>(already Minted)</span>;
+  else if (operator == userAddress) {
+    showRegister = true;
+    if (signerAddress == operator)
+      registerMessage = <span>(operator can't register)</span>;
```

Comment:
Must use the escaped apostrophe, otherwise the build also fails.
```suggestion
      registerMessage = <span>(operator can&apos;t register)</span>;
```